src/ai/backend/plugin/BUILD

```diff
@@ -18,7 +18,10 @@ visibility_private_component(
         "//src/ai/backend/testutils/**",
         "//src/ai/backend/install/**",
     ],
-    allowed_dependencies=[],
+    allowed_dependencies=[
+        "//src/ai/backend/cli/**",
+        "//src/ai/backend/accelerator/**",
```

Comment:
During the build step, we don't ship the plugin subsystem with the accelerator plugins included. The accelerators should be loaded dynamically at runtime, not in the build-time.